.github/workflows/repo_fetcher.yml

```
@@ -0,0 +1,34 @@
+name: Repo Contributions Fetcher
+
+on:
+  workflow_dispatch:  
+
+jobs:
+  fetch_issues_prs:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout Repo
```

Fail fast on missing PAT.

- name: Check for GH_TOKEN
  run: |
    if [ -z "${GH_TOKEN}" ]; then
      echo "::error::GH_TOKEN is not set"
      exit 1
    fi